os/include/tinyara/mpu.h

```diff
@@ -49,4 +49,27 @@ enum {
 #define STACKGUARD_MPU_REGION_NUM		3	/* Region number 3 is reserved for stack protection */
 #endif
 
+/* Enum for MPU region numbers */
+
+enum MPU_REG_NUM {
```

Comment:
This shows hard-coded mpu region. I don't know what you do in this PR.

Reply:
Sangam has informed me that as per latest discussions, this type of implementation is fine